app/controllers/hyrax/batch_edits_controller.rb

```diff
@@ -135,5 +154,15 @@ def redirect_to_return_controller
         redirect_to hyrax.dashboard_path
       end
     end
+
+    # Clean up form fields
+    # @param form Hyrax::Froms::ResourceBatchEditForm
+    def cleanup_form_fields(form)
```

Comment:
Suggestion: Can we get a comment about why this is necessary? Why doesn't the form handle this somewhere? Does this mean the form's not useful outside of this controller?

Reply:
i'm also curious about this.
it does seem like the form should handle these transformations.

Reply:
I am not sure what is the best way to handle it. The another way that we can try is to merge the submitted params to the resource attributes in [form.validate(params)](https://github.com/samvera/hyrax/pull/6148/files#diff-7eea01c84c5eba32121b213d71b5dd9f86b7968181dd4a273348d42e00463bd3R65). Please advice.

Reply:
Action item: Fix this, I can take a look at it